packages/plugin-ext/src/main/browser/theme-icon-override.ts

```diff
@@ -0,0 +1,220 @@
+// *****************************************************************************
+// Copyright (C) 2022 TypeFox and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { ThemeIcon } from '@theia/monaco-editor-core/esm/vs/platform/theme/common/themeService';
+
+// Keep this up-to-date with the table at https://code.visualstudio.com/api/references/icons-in-labels#icon-listing
```

Comment:
Monaco uplifts seem like a reasonable time to do this, so let's mark it like this to get it on the agenda the next time someone does one (if they read [my docs](https://github.com/eclipse-theia/theia/blob/master/packages/monaco/README.md))
```suggestion
// @monaco-uplift
// Keep this up-to-date with the table at https://code.visualstudio.com/api/references/icons-in-labels#icon-listing
```